### DISCUSSION TITLE
IP Adapter, Instant ID - help

### DISCUSSION THREAD

**Discussion by Joosheen** [lllyasviel/stable-diffusion-webui-forge#432] [Q&A]
Hey,

in Forge there are other preprocessors (for example there is inside_id_face_embeddings). 
So do you use IP Adapter and Instant ID in Forge and if the answer is yes, can you tell me (or us) how to make proper settings?

Regards, A.J.

**Comment by ovladuk** [comment]
have you figured out how to use it? i can't get it to work

**Comment by calberts** [comment]
Any solution for this still no answer for this it seems a Forge bug???

**Comment by lllyasviel** [comment]
https://github.com/lllyasviel/stable-diffusion-webui-forge/discussions/178#discussioncomment-8572388

**Reply by Wootbloot** [reply]
this doesn't solve the issue
